Rock/MergeTemplates/WordDocumentMergeTemplateType.cs

```diff
@@ -73,6 +73,8 @@ public override BinaryFile CreateDocument( MergeTemplate mergeTemplate, List<obj
             BinaryFile outputBinaryFile = null;
 
             var rockContext = new RockContext();
+            rockContext.Database.CommandTimeout = 600;
+
```

Comment:
same thing here, this shouldn't be necessary, getting a BinaryFile by Id should take less than a second